docs/manual/cli/automator.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "contao:automator"
+description: "Common tasks around the maintenance of a Contao installation."
+aliases:
+    - /en/cli/automator/
+weight: 15    
+---
+
+
+This command is an interface to the `Automator` class of Contao. It mainly consists
+of common tasks around the maintenance of a Contao installation.
+
+```bash
+php vendor/bin/contao-console contao:automator [<task>]
+```
+
+The command will prompt you for which task should be executed, if no specific task is provided. The following tasks are available.
+
+| Task                    | Description                                                                                                                                           |
+|-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `purgeSearchTables`     | Clears the search index by deleting the tables `tl_search` and `tl_search_index`.                                                                     |
+| `purgeUndoTable`        | Removes the temporarily stored undo entries, which allows for restoring deleted entries.                                                              |
+| `purgeVersionTable`     | Purges the version table `tl_version` which holds different versions for rows in tables that have the versioning feature enabled.                     |
+| `purgeSystemLog`        | Deletes entries in the system log.                                                                                                                    |
+| `purgeImageCache`       | Purges the image cache by deleting all processed and resized images in the configurable `image.target_dir`.                                           |
+| `purgeScriptCache`      | Removes processed and/or minified JavaScript files and Stylesheets.                                                                                   |
+| `purgePageCache`        | Clears the page cache by removing cached html responses.                                                                                              |
+| `purgeSearchCache`      | Clears the search result cache.                                                                                                                       |
+| `purgeInternalCache`    | Clears the internal cache directory in `kernel.cache_dir`, this includes the Symfony application cache and potentially hits non Contao caches.        |
```

Comment:
```suggestion
| `purgeInternalCache`    | Clears the internal cache directory in `kernel.cache_dir`, this includes the Symfony application cache and potentially hits non Contao caches.        |
```

Where did you pull this information from?

Reply:
From https://docs.contao.org/dev/reference/commands/#contao-automator

Reply:
I think this is wrong, it should be:

```suggestion
| `purgeInternalCache`    | Clears Contao's internal cache for DCA and language files. |
```